README.md

```diff
@@ -98,4 +100,38 @@ $ make test-bank-supply-of
 ```
 $ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"bank_supplyOf","params":{"token_address":"sov1zdwj8thgev2u3yyrrlekmvtsz4av4tp3m7dm5mx5peejnesga27svq9m72"},"id":1}' http://127.0.0.1:12345
 {"jsonrpc":"2.0","result":{"amount":1000},"id":1}
-```
\ No newline at end of file
+```
+
+#### Optional: Setup the gas constants for transaction execution.
```

Comment:
This section on gas has useful information but seems out of proportion to the rest of the README (It's as long as the rest of the README put together). Can we pare it down to just... 
- A super bare-bones getting started guide for gas (i.e. set up your account with _ tokens, set your max_fee to _)
- A pointer to a more full explanation